python/ray/dataframe/dataframe.py

```diff
@@ -72,6 +75,12 @@ def _default_index(self):
 
     index = property(_get_index, _set_index)
 
+    def _update_lengths(self):
```

Comment:
Prefer name change to `_compute_lengths()`. We have a set and the name doesn't sound much different.